app/konnect/mesh-manager/federate-zone.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Federate a zone control plane to {{site.konnect_saas}}
+content_type: tutorial
+---
+
+If you already have a zone control plane which is not connected to any global control plane, you can federate it to {{site.konnect_short_name}} in [Mesh Manager](/konnect/mesh-manager/). 
+
+By federating a zone control plane, you move {{site.mesh_product_name}} from a single-zone setup to [multi-zone](/mesh/latest/production/deployment/multi-zone/) setup. One benefit of moving from single-zone to multi-zone is it gives you automatic service fail-over in case a specific zone fails.
+
+This guide explains how to federate a zone control plane to {{site.konnect_short_name}} by migrating an existing {{site.mesh_product_name}} zone to {{site.konnect_short_name}} and adding another zone.
+
+## Prerequisites
+
+* A universal or Kubernetes cluster with a running zone control plane that isn't connected to the global control plane in {{site.konnect_short_name}}
+* [The latest version of `kumactl`](/mesh/latest/production/install-kumactl/) installed and configured to communicate with the self-managed global control plane
+* A [{{site.mesh_product_name}} global control plane in {{site.konnect_short_name}}](/konnect/mesh-manager/service-mesh/#create-a-zone-in-the-global-control-plane)
+
+## Transfer resources from the zone control plane to {{site.konnect_short_name}}
```

Comment:
@jakubdyszkiewicz One thing I thought of is are we migrating the Kong Mesh zone control plane to Konnect here? If so, do you think it would make sense to link these migration instructions (https://docs.konghq.com/konnect/mesh-manager/migrate-zone/) or even make this content part of that migrate zone page? I was thinking it could maybe go in a "multi-zone" section on that page.

Reply:
It's a bit tricky. The page you linked is about moving a zone control plane connected to self-managed global control plane to cloud-managed global control plane.

This PR is about moving a zone control plane not connected to any global control plane to cloud-managed global control plane.